packages/experimental-nextjs-app-support/README.md

```diff
@@ -154,6 +158,60 @@ export default function RootLayout({
 
 If you want to make the most of the streaming SSR features offered by React & the Next.js App Router, consider using the [`useSuspenseQuery`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usesuspensequery_experimental) and [`useFragment`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usefragment_experimental) hooks.
 
+### Preloading data in RSC for usage in Client Components
+
+You can also preload data in RSC to populate the cache of your Client Components.
+
+For that, follow the setup steps for both RSC and Client Components as laid out in the last two paragraphs.
```

Comment:
```suggestion
As a prerequisite, follow the steps outlined in the ["In RSC"](#in-rsc) and ["In Client Components and streaming SSR"](#in-client-components-and-streaming-ssr) sections to configure your client.
```

Alternatively, you can reference the section names with links to really make sure we know what to follow here :)

Reply:
Those links will be dead with the first rewording PR, so I'll choose the version without links :D